sympy/core/power.py

```diff
@@ -1533,7 +1533,8 @@ def _eval_nseries(self, x, n, logx, cdir=0):
                 return res
 
         f = b.as_leading_term(x, logx=logx)
-        g = (b/f - S.One).cancel(expand=False)
+        g = (b.expand() - f).cancel()
```

Comment:
What type of expansion is needed here?

Reply:
I too tried to use other wrappers of expand like expand_multinomial(), the thing is that we need to ensure that we are able to write b in a way such that each individual term of its expansion is exposed among them the leading nonzero term too, expand() works for this example for simple expansion of `(x - 0.23)**4`, perhaps we should use something like 
```
 g = (b.series(x, n=2).removeO() - f).cancel()
```

Reply:
Yeah not fully clear as to if we need expand there and that too what sort of expansion :|

Reply:
use `_mexpand` to get a flat sum or terms